cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -53,7 +55,18 @@ object CqlParser extends JavaTokenParsers
    */
   def constant: Parser[Constant] = {
     import Constants._
-    (string | uuid | number | boolean) ^^ Constant // | hex // TODO
+    def str = string ^^ StringConstant
```

Comment:
Most of parsers defined in Constants object are only used here. Maybe you can cut/paste them here instead of having dummy variable names like "string" and "str".